pkg/ipamd/ipamd.go

```diff
@@ -1731,6 +1737,13 @@ func enablePodENI() bool {
 	return utils.GetBoolAsStringEnvVar(envEnablePodENI, false)
 }
 
+func getNetworkPolicyMode() string {
+	if value := os.Getenv(envNetworkPolicyMode); value != "" {
```

Comment:
nit: This should either log an error on a typo like "strct", or we should convert to a boolean or enum

Reply:
yes will be adding few logs around this and we would need aws-node to fail..